### ISSUE TITLE
Alignment issues

### ISSUE THREAD

**Issue by iRoachie** [umhan35/react-native-search-bar#104]
The icon and the placeholder isn't aligned.
![screen shot 2016-12-24 at 6 03 55 am](https://cloud.githubusercontent.com/assets/5962998/21466356/e72cb838-c99e-11e6-854b-b80e97376dca.jpg)

"react-native-search-bar": "^2.16.0"
"react-native": "0.39.2"

Problem seems to occur when using multiple search bars in a project.

**Comment by tmaximini** [comment]
did you find a solution for this?

**Comment by iRoachie** [comment]
Nah Man. I haven't got the slightest clue

**Comment by JodiWarren** [comment]
While I couldn't actually fix this one I did find a workaround. It seemed to happen when I had any of the colour properties set, i.e. `barTintColor`. So I set `hideBackground={true}` and set `backgroundColor` on the parent View, and this seemed to work around the bug effectively.

Before: 
```jsx
<View>
    <SearchBar barTintColor="#f00" />
</View>
```

After:
```jsx
<View style={{backgroundColor: '#f00'}}>
    <SearchBar hideBackground={true} />
</View>
```

**Comment by tmaximini** [comment]
@JodiWarren thanks, but removing barTintColor property or adding hideBackgroundColor does not affect alignment of placeholder text or Icon at all in my case, it still looks of like in the screenshot above.

**Comment by JodiWarren** [comment]
@tmaximini

This is literally the code I'm using in my render method. Give it a shot, maybe it'll help find out what's different about our setups:

```
<View style={{
    marginTop: 20,
    backgroundColor: '#ccc'
}}>
    <SearchBar
        ref={(bar) => {this.searchBar = bar}}
        onFocus={this._onFocus}
        onCancelButtonPress={this._onCancelButtonPress}
        onSearchButtonPress={this._onSearchButtonPress}
        placeholder="Search Products"
        showsCancelButton={this.state.showsCancelButton}
        hideBackground={true}
    />
</View>
```

Here's the end result:
![image](https://cloud.githubusercontent.com/assets/1337003/23959028/dd5e16c0-099b-11e7-9384-bb0a991f6009.png)

**Comment by iRoachie** [comment]
Can you try using multiple search bars? Like on different pages? I only encountered the problem when there were more than one

**Comment by JodiWarren** [comment]
@iRoachie I just chucked a few extra components in, and it works for me too:

Same page:
![image](https://cloud.githubusercontent.com/assets/1337003/23960520/17fb867e-09a0-11e7-8495-9f89721938f7.png)

Different page:
![image](https://cloud.githubusercontent.com/assets/1337003/23960530/24b2014a-09a0-11e7-9c32-ed6e027ea4d6.png)

If you've removed all the props that I'm not using and you're not getting the same result as me, then there must be something else interfering - have you tried it in a new toy app, just to see?

**Comment by iRoachie** [comment]
Can you try changing the color of the text input and tell me if it still behaves the same?

**Comment by JodiWarren** [comment]
@iRoachie I did mention in my earlier comment (https://github.com/umhan35/react-native-search-bar/issues/104#issuecomment-284012858) setting any of the colour props like `textColor` would cause this issue. Is that what you mean?

**Comment by JodiWarren** [comment]
Btw, I have found that this glitch is especially sticky when you're in the simulator - I sometimes have to kill the React Native packager and iOS simulator, run `killall node` in the command line, and then restart it to get the changes to appear.

**Comment by iRoachie** [comment]
It happened to me on device as well.

**Comment by iRoachie** [comment]
Anyway you think we could style input, and without affecting the layout

**Comment by JodiWarren** [comment]
Not without someone with more time and Objective-C knowledge than me fixing the actual bug! I'm just proposing a way to work around it, if all you need is to change the background colour. If you need to change the text colour, then I'm afraid that I can't help.

**Comment by tmaximini** [comment]
@JodiWarren what versions of iOS, react-native and this package are using? I have the alignment issue even when just using the bare default, without any background colors.

**Comment by lukemiles** [comment]
I fixed this in #121 (could only reproduce when color of text was changed)
